src/invoices/InvoicesList/InvoicesListFilters/FiscalYearFilter/FiscalYearFilter.js

```diff
@@ -0,0 +1,32 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Loading } from '@folio/stripes/components';
+import { SelectionFilter } from '@folio/stripes-acq-components';
+import { usePayableFiscalYears } from '../../../../common/hooks';
+
+export const FiscalYearFilter = (props) => {
+  const { isLoading, fiscalYears } = usePayableFiscalYears();
```

Comment:
filter will display only `PayableFiscalYears`, but filter should allow to filter by any FYs